Tooltips: Consistency and contrast: Issue #11180

timwright12 opened the pull request:
## Description
Tooltips: Consistency and contrast issues
Fixes https://github.com/WordPress/gutenberg/issues/11180

## How has this been tested?
Tested in the browser, precommit hooks and e2e tests.

## Screenshots
<img width="264" alt="screen shot 2018-11-16 at 3 44 31 pm" src="https://user-images.githubusercontent.com/459581/48646450-e4fe7080-e9b6-11e8-9ebc-e567520ea4cf.png">
<img width="277" alt="screen shot 2018-11-16 at 3 44 58 pm" src="https://user-images.githubusercontent.com/459581/48646451-e4fe7080-e9b6-11e8-81a8-e65074e49037.png">
<img width="321" alt="screen shot 2018-11-16 at 3 45 03 pm" src="https://user-images.githubusercontent.com/459581/48646452-e4fe7080-e9b6-11e8-8c3e-27eaa825d60a.png">


## Types of changes
Introduces a prop to tooltips passed through iconButtons to control the label positioning.

## Checklist:
- [ x ] My code is tested.
- [ x ] My code follows the WordPress code style.
- [ x ] My code follows the accessibility standards.
- [ x ] My code has proper inline documentation.

Commit:
merge from upstream

Commit:
Merge branch 'master' of github.com:WordPress/gutenberg

Commit:
adjusted editor header tooltip label positions for better contrast

Commit:
Merge branch 'master' of github.com:WordPress/gutenberg

timwright12 committed:
Merge branch 'master' into fix/issue-11180

Commit:
Merge branch 'master' of github.com:WordPress/gutenberg

Commit:
merge from master

Commit:
Merge branch 'master' of github.com:WordPress/gutenberg

Commit:
Merge branch 'master' into fix/issue-11180

Commit:
merge from upstream

Commit:
Merge branch 'master' of github.com:WordPress/gutenberg

Commit:
Merge branch 'master' of github.com:WordPress/gutenberg

Commit:
Merge branch 'master' into fix/issue-11180

Commit:
Merge branch 'master' of github.com:WordPress/gutenberg

Commit:
Merge branch 'master' of github.com:WordPress/gutenberg

Commit:
Merge branch 'master' into fix/issue-11180

Commit:
Merge branch 'master' of github.com:WordPress/gutenberg

Commit:
Merge branch 'master' of github.com:WordPress/gutenberg

tofumatt reviewed (changes requested):
This looks great, actually! Could you run the snapshot test update (using `npm run test-unit:update`) to get the tests passing, and also remove the `gutenberg-mobile` git submodule?

After that I think this would be good to merge, though it'd be great if you could add some screenshots as well. 😄

Commit:
Merge branch 'master' of github.com:WordPress/gutenberg

Commit:
Merge branch 'master' into fix/issue-11180

Commit:
removed gutenberg mobile

Commit:
udpated snapshot

timwright12 commented:
@tofumatt totally forgot this one existed, nice pick up! I removed the `gutenberg-mobile` submodule, updated the snapshot and added some screenshots to the ticket.

Let me know if there's anything else! Thanks again!

tofumatt reviewed (approved):
Awesome, thanks! 👍
